### ISSUE TITLE
Future of this plugin development

### ISSUE THREAD

**Issue by unfulvio** [unfulvio/wp-api-menus#60]
Hey there,

I'm sorry if I have unanswered many issues opened here, or support requests on WordPress.org. 

It's a shame, I just don't have the time to maintain the plugin. 

In the past few years I've added some collaborators to this project:

* @fahrradflucht
* @mckernanin
* @robertsky
* @thefrosty (just today)

Please let me know if you're still actively monitoring this project as collaborators and if you'd like to coordinate with each other. 

If there's interest, I would recommend that the project has a coordinator or a lead. So I'd advise for some of you to commit to that role than just patching the plugin.

Eventually this person could also get authorship on WordPress.org and publish updates moving forward.

Cheers and thanks for having shown interest and support for a plugin born out of a side project to support some personal/work projects I had back then several years ago! :)

**Comment by fahrradflucht** [comment]
I'm not using this plugin anymore nor have I written serious WordPress code in ages. I'm probably not a good fit to move this forward.

**Comment by thefrosty** [comment]
I can say that myself and my company are using this, which is why I've asked for contributor access. I would love to get .org access too, just let me know what you would like to see before that it completed.

I can also say that with https://core.trac.wordpress.org/ticket/40878 in flight for WordPress 5.6, this repo and the .org one can be safely deprecated by this time next year (maybe). So contributor support could be limited to just 5.5's breaking changes.

👋

**Comment by unfulvio** [comment]
Thanks! I am actually surprised that this feature (menu support in the REST API) didn't land in core for so so long. That wouldn't be the longest thing in WordPress, but given all the attention that REST/JS had in the recent years is the surprising bit. 

Back then I was developing single-page apps/websites with a frontend/theme in JS and I needed menu endpoints. I never focused on this plugin afterwards, I always thought it was meant to be short lived and not working any longer with themes I didn't feel I could drive its development by fist hand experiences. 

If you submit a PR with all the changes and test the plugin, then you could edit the readme assets and add your name there. Please update stable tag, version number, etc. Then, I should be able to issue a SVN update (fingers crossed, I don't even know the deploy scripts still work lol) and ad that point you'd also have SVN commit rights.

**Comment by thefrosty** [comment]
Opened the PR for version 1.3.2 #61

**Comment by thefrosty** [comment]
Looks like the packagist is tied to your old GitHub handle, looking into it a bit more: https://packagist.org/packages/nekojira/wp-api-menus

**Comment by mckernanin** [comment]
I am also not working in WP anymore.

**Comment by thefrosty** [comment]
This only needs to be supported for the next ~four months until it's exposed in core.

**Comment by unfulvio** [comment]
Alright thanks @thefrosty I've published the new version on WordPress.org - thanks for your help. Perhaps when things are stable we can issue a final version to warn about deprecation / invite people to switch and then and retire the plugin later on.

**Comment by thefrosty** [comment]
Absolutely, I'll check in on things as we inch closer to menus in core.

**Comment by acafourek** [comment]
@unfulvio my team is actively using a forked version of the plugin on a few production sites and I'd love to see it continue until we have core support. It looks like WP ticket 40878 has been pushed to "Future Release" since mid-2020.

I'd be happy to help maintain going forward - will also clean up our forked version and open a PR to add support for specifying which menu IDs to return from the endpoint

**Comment by unfulvio** [comment]
hey @acafourek thanks for offering. I have not been following up with WP core dev WRT menu items. @thefrosty are you still also working actively on this plugin in the meantime support is added natively? Just mentioning you here in case you'd want to have @acafourek as another collaborator on the project and if you guys would like teaming up. Thanks

**Comment by thefrosty** [comment]
I am actively using the plugin, but haven't needed any _new_ features. We have a modification externally to filter "items" in the /menus response but not a need to make any updates currently for support until core adopts menus.

**Comment by thefrosty** [comment]
> After nearly two and half years of work, the menu, menu item and menu locations REST API endpoints are finally merged into WordPress core. https://core.trac.wordpress.org/changeset/52079

**Comment by Saggitarie** [comment]
When is this going to be released?

**Comment by jwanner83** [comment]
@Saggitarie the milestone, according to this issue (https://core.trac.wordpress.org/ticket/40878), is 5.9, but they are still fixing some bugs in the code as you can see in the comments.

**Comment by thefrosty** [comment]
WordPress 5.9 Menus endpoints: https://make.wordpress.org/core/2022/01/24/menus-endpoints-in-wordpress-5-9/

**Comment by thefrosty** [comment]
https://make.wordpress.org/core/2022/01/07/the-new-navigation-block/
